namespace TestTryCatchTwoExceptionsFinally is
    use Std = IO.Std;

    class Main is
        entry() static is
            @IL.output("il.out")
            try
            // FIXME: ex1 and ex2 should be in their
            // own scopes, per catch block
            catch ex1: System.Exception
            catch ex2: MyException
            finally
            yrt
        si
    si

    class MyException: System.Exception is
        
    si
    
si